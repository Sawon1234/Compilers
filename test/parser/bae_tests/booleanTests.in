function testAnd(Boolean a, Boolean b) : Boolean
{
	return a&b;
}

function testOr(Boolean a, Boolean b) : Boolean
{
	return a|b;
}