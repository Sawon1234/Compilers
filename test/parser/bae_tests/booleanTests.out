function testAnd ( Boolean a , Boolean b) : Boolean { return a and b ; } function testOr ( Boolean a , Boolean b ) : Boolean { return a or b ; }